<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BLRLAP653" ignored="3" name="Default test(failed)" tests="6" failures="2" timestamp="09 May 2018 05:39:18 GMT" time="0.141" errors="0">
  <testcase name="Test1" time="0.032" classname="RerunFailedTest.Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.004" classname="RerunFailedTest.Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.003" classname="RerunFailedTest.Test001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at RerunFailedTest.Test001.Test1(Test001.java:14)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.003" classname="RerunFailedTest.Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.003" classname="RerunFailedTest.Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.003" classname="RerunFailedTest.Test001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at RerunFailedTest.Test001.Test2(Test001.java:21)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase name="Test4" time="0.0" classname="RerunFailedTest.Test001">
    <ignored/>
  </testcase> <!-- Test4 -->
  <testcase name="Test3" time="0.0" classname="RerunFailedTest.Test001">
    <ignored/>
  </testcase> <!-- Test3 -->
  <testcase name="Test5" time="0.0" classname="RerunFailedTest.Test001">
    <ignored/>
  </testcase> <!-- Test5 -->
</testsuite> <!-- Default test(failed) -->
